static/src/stylesheets/module/content/_article-immersive.scss

```diff
@@ -237,6 +237,19 @@
 
     /* Meta
     ========================================================================== */
+    .section-label--advertisement {
+        @include fs-textSans(3);
```

Comment:
Seems silly to use this mixin, and then overwrite the `font-size` and `line-height`. Can you either add another line to this mixin [here](https://github.com/guardian/frontend/blob/master/static/src/stylesheets/pasteup/typography/_src.scss#L42) and name it 7, or don't use this mixin, and use this one instead: https://github.com/guardian/frontend/blob/master/static/src/stylesheets/pasteup/typography/_src.scss#L114